src/api/utils.go

```diff
@@ -126,6 +127,17 @@ func JsonHandler(f HandlerFunc) errorFunc {
 	}
 }
 
+// Adds CORS headers to a response.
+func CORS(w http.ResponseWriter, methods ...string) {
+	w.Header().Set("Accept", "application/json")
+	w.Header().Set("Access-Control-Allow-Origin", "*")
+	w.Header().Set("Access-Control-Allow-Headers", "Content-Type")
+	w.Header().Del("Access-Control-Allow-Methods")
+	for _, s := range methods {
+		w.Header().Add("Access-Control-Allow-Methods", s)
```

Comment:
you can add this header only once: `Access-Control-Allow-Methods: POST, GET, OPTIONS`

see https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS#Access-Control-Allow-Methods